x/cronos/keeper/keeper_test.go

```diff
@@ -354,24 +354,11 @@ func (suite *KeeperTestSuite) TestRegisterOrUpdateTokenMapping() {
 			},
 			false,
 		},
-		{
-			"Source token, denom not match",
-			types.MsgUpdateTokenMapping{
-				Sender:   "",
-				Denom:    "cronos0xA6d4fecb1a6fb7c2ca350169a050d483bd87b883",
-				Contract: contractAddress,
-				Symbol:   "",
-				Decimal:  0,
-			},
-			func() {
-			},
-			true,
-		},
 		{
 			"Source token, denom not checksum, error",
```

Comment:
might change desc now, since it's invalid contract error

Reply:
changed